apps/reference/docs/guides/auth/auth-mfa.mdx

```diff
@@ -0,0 +1,570 @@
+---
+id: auth-mfa
+title: 'Multi-Factor Authentication'
+description: Add an additional layer of security to your apps with Supabase Auth multi-factor authentication.
+---
+
+import { Mermaid } from 'mdx-mermaid/Mermaid';
+
+:::note
+Multi-Factor Authentication is in early access preview only.
+:::
+
+Multi-factor authentication (MFA) or sometimes also called two-factor
+authentication (2FA) is a way to add an additional layer of security to your
+application by requiring users to verify their identity through one or more
+additional verification steps.
+
+It is considered a best practice for all but the simplest of applications.
```

Comment:
```suggestion
It is considered a best practice to use MFA for your applications.
```

Reply:
I think "best practice" implies "some exceptions"? Perhaps we don't need to add the type of exceptions